eng/common/matrix/functions.ps1

```diff
@@ -0,0 +1,345 @@
+Set-StrictMode -Version "4.0"
```

Comment:
I know this is scoped to a folder but I'd still call this something more specific then functions.